requirements.txt

```diff
@@ -1 +1,7 @@
-streamlit
\ No newline at end of file
+streamlit
+streamlit-audiorecorder
+openai-whisper
```

Comment:
For all packages in `requirement.txt`, better to specify the exact version that we are using